doc/releasenotes/17_0_0_summary.md

```diff
@@ -3,11 +3,66 @@
 ### Table of Contents
 
 - **[Major Changes](#major-changes)**
+  - **[Breaking Changes](#breaking-changes)**
   - **[New command line flags and behavior](#new-flag)**
     - [Builtin backup: read buffering flags](#builtin-backup-read-buffering-flags)
 
 ## <a id="major-changes"/> Major Changes
 
+### <a id="breaking-changes"/> Breaking changes
+
+#### Dedicated stats for VTGate Prepare operations
```

Comment:
This note is too long. Can it be condensed? GH imposes a limit on how much content you can have on a release. We exceeded that for v16-rc1.

Reply:
I would not call this a breaking change. It is a bug fix. Yes people's metrics will show different numbers if they are using Prepare, but they won't become non-functional.

Reply:
I won't argue you about naming, but conceivably this could break alerts for people if they make heavy use of prepared statements and are relying on Execute metrics to indirectly monitor them. People using Vitess might expect to find this kind of information in a breaking changes section, even if we (you and other maintainers) don't think of it that way. 

Will change it as you've suggested, just offering my 2c!

Reply:
You make some good points. Yes, let's keep it in the breaking changes section.